observability-kit-test/observability-kit-agent-it/src/test/java/com/vaadin/observability/test/AbstractViewIT.java

```diff
@@ -32,12 +30,12 @@ abstract class AbstractViewIT extends BrowserTestBase
     public static void setupClass() {
         String hubHost = Parameters.getHubHostname();
         isHub = hubHost != null && !hubHost.isEmpty();
-        if (!isHub) {
-            String driver = System.getProperty("webdriver.chrome.driver");
-            if (driver == null || !new File(driver).exists()) {
-                WebDriverManager.chromedriver().setup();
-            }
-        }
+        /*
+         * if (!isHub) { String driver =
+         * System.getProperty("webdriver.chrome.driver"); if (driver == null ||
+         * !new File(driver).exists()) {
+         * WebDriverManager.chromedriver().setup(); } }
+         */
```

Comment:
Can this commented code be removed?

Reply:
Yes, I forgot to clean up. Thanks for spotting

Reply:
done